core/src/empty/empty.module.css

```diff
@@ -0,0 +1,14 @@
+.container {
+    display: flex;
```

Comment:
The spacing here seems wrong. Moai uses tab, not 4 spaces. Please make sure you have https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode installed and run it against your new files